### DISCUSSION TITLE
Some improvements ideas about the timeline navigation

### DISCUSSION THREAD

**Discussion by ValdoTR** [community/community#64510] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi,
I've been using my Github profile page for a while now (to find out what I've been doing for the past month) and I feel that the interface could be greatly improved with just 2 minor modifications:

1. White border: to see what is the currently selected day on the grid
![feature github -  highlight day](https://github.com/community/community/assets/14997703/32d4b56f-7267-48ee-9f36-93b51e6d9d1e)

2. Prev / next arrows: to navigate through the days without using the grid (faster, and prevent miss clicks)
![feature github -  next day](https://github.com/community/community/assets/14997703/330da9a2-aa08-4d6e-983b-21093d2cf5cc)

**Comment by Rockyread** [comment]
That's great
